smart-frontend/app/src/store/adh/rules/rulesSlice.ts

```diff
@@ -75,10 +77,42 @@ const refreshRules = createAsyncThunk('adh/rules/refreshRules', async (_, thunkA
   thunkAPI.dispatch(loadRules());
 });
 
+const getActiveRulesCount = createAsyncThunk('adh/rules/getActiveRulesCount', async (_, thunkAPI) => {
```

Comment:
I think we can combine this thunk actions to one action `getRulesMetaInfo`
And in this method send two request to api   `AdhRulesApi.getRules({ ruleStates: [AdhRuleState.Active] });` and ` AdhRulesApi.getRules({});`
One method - one extrareducer.

Reply:
`getRulesMetaInfo` added